sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java

```diff
@@ -1494,17 +1471,70 @@ public Mono<String> getAudioTranscriptionAsPlainText(
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return chat completions for the provided chat messages.
+     * This is an Azure-specific version of chat completions that supports integration with configured data sources and
+     * other augmentations to the base chat completions capabilities on successful completion of {@link Mono}.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    Mono<ChatCompletions> getChatCompletionsWithAzureExtensions(String deploymentOrModelName,
+        ChatCompletionsOptions chatCompletionsOptions) {
+        // Generated convenience method for getChatCompletionsWithAzureExtensionsWithResponse
+        RequestOptions requestOptions = new RequestOptions();
+        return getChatCompletionsWithAzureExtensionsWithResponse(deploymentOrModelName,
+            BinaryData.fromObject(chatCompletionsOptions), requestOptions).flatMap(FluxUtil::toMono)
+                .map(protocolMethodData -> protocolMethodData.toObject(ChatCompletions.class));
+    }
+
+    /**
+     * Starts the generation of a batch of images from a text caption.
+     *
+     * @param imageGenerationOptions Represents the request data used to generate images.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the {@link PollerFlux} for polling of a polling status update or final response payload for an image
+     * operation.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
+    PollerFlux<BatchImageGenerationOperationResponse, BatchImageGenerationOperationResponse>
+        beginBeginAzureBatchImageGeneration(ImageGenerationOptions imageGenerationOptions) {
+        // Generated convenience method for beginBeginAzureBatchImageGenerationWithModel
+        RequestOptions requestOptions = new RequestOptions();
+        return serviceClient.beginBeginAzureBatchImageGenerationWithModelAsync(
+            BinaryData.fromObject(imageGenerationOptions), requestOptions);
+    }
```

Comment:
Yeah.  Since they are not public APIs it doesn't block us. We need to replace the LRO ImageGeneration APIs by the non-LRO/blocking ImageGeneration APIs, Let's clean this API learn if needed.

Reply:
They are the method that marked as `@@access(, internal)` in client.tsp. So it is generated as "package private".

I'd like to actually add `@@convenientApi(false)` to them. But sadly `@convenientApi` decorator does not have scope, so I cannot do it for Java alone.